WebRoot/skins/_base/base4/skin.css

```diff
@@ -758,6 +758,14 @@ we are hiding element with that class
     fill: @IconCheckboxUnchecked@;
 }
 
+.ZTreeItemCheckbox .ImgCheckboxChecked-svg .svg-icon {
```

Comment:
Updated the rules as per offline discussion:
0e0e88c9e3034ae1b51f6657d6aaf2d93a0c4a03